cmd/bucket-handlers.go

```diff
@@ -477,6 +485,15 @@ func (api objectAPIHandlers) PutBucketHandler(w http.ResponseWriter, r *http.Req
 	vars := mux.Vars(r)
 	bucket := vars["bucket"]
 
+	objectLockEnabled := false
+	if _, found := r.Header[textproto.CanonicalMIMEHeaderKey("x-amz-bucket-object-lock-enabled")]; found {
+		if trueString := r.Header.Get("x-amz-bucket-object-lock-enabled"); trueString != "true" {
```

Comment:
This can be one line right? 

```suggestion
		if r.Header.Get("x-amz-bucket-object-lock-enabled") != "true" {
```

Reply:
done